templates/nginx_internal.conf

```diff
@@ -57,15 +57,10 @@ http {
 
         gzip on;
         gzip_types text/css application/javascript image/svg+xml;
-        gzip_vary on;
```

Comment:
You can get rid of the other gzip lines too as the other nginx config already has them.

Reply:
bump

Reply:
That shouldn't cause any issues, but I can remove it shortly once I'm done with $dayjob. Thanks for the bump @Nutomic